File: JAVA/Main2.java
package JAVA;

public class Main2{
    public static void main(String[] args){
        // 🔔variable = a reusable container for a value
        //            a variable behaves as if it was the value it contains

        // 📕Primitive = simple value stored directly in memory (stack)
        // 📘Reference = memory address (stack) that points to the (heap)

        // 📕Primitive vs 📘Reference
        // -----------    ---------
        // int            string
        // double         array
        // char           object
        // boolean

        int age; // declaration
        age = 8; // assignment
        System.out.println(age);

        double salary;     // declaration
        salary = 75000.00; // assignment
        System.out.println(salary);

        char grade;  // declaration
        grade = 'A'; // assignment
        System.out.println(grade);

        boolean isStudent;
        isStudent = true;
        if (isStudent){
            System.out.println("You are a student");
        }
        else{
            System.out.println("You are not a student");
        }

        String name = "Bro Code";
        System.out.println(name + " is a Legend");

        int arr[] = {0, 9, 8, 7};
        for (int i = 0; i < arr.length; i ++){
            System.out.print(arr[i] + " ");
        }

    }
}

File: Python/main2.py
# variable = A container for a value (string, integer, float, boolean)
#            A variable behaves as if it was the value it contains


#String
first_name = "bro"
last_name = "code"
fav_food = "pizza"
email = "bro123@gmail.com"

print(f"hello {first_name} {last_name}")
print(f"You like {fav_food}")
print(f"your email is {email}")

#Integer

age = 25
quantity = 3
numOfStudents = 110

print(f"You are {age} years old")
print(f"you bought a total of {quantity} {fav_food}s")
print(f"your class has {numOfStudents} students")

#Float

price = 10.99
gpa = 7.3
distance = 5.5

print(f"the price of a {fav_food} is {price}")
print(f"your gpa is {gpa}")
print(f"you ran over {distance} miles")

#Boolean

is_student = True
is_homophobic = True
is_online = False

if is_student:
    print("you are a student")
else:
    print("you are NOT a student")

if is_homophobic:
        print("pd is homophobic")
else:
        print("pd is gay")

if is_online:
    print("you are online")
else:
    print("you are not online")